dsgnwrks-code-snippets-cpt.php

```diff
@@ -105,6 +208,28 @@ public function add_languages() {
 		}
 	}
 
+	public function version_check() {
+		$current_version = get_option( 'dsgnwrks_snippetcpt_version' );
+		if ( self::VERSION !== $current_version ) {
+			$this->update();
+		}
+	}
+
+	function update() {
+		$terms = get_terms( 'languages', array(
+			'fields'     => 'ids',
+			'hide_empty' => false,
+		) );
+
+		if ( ! is_wp_error( $terms ) ) {
+			foreach ( $terms as $term ) {
+				wp_delete_term( $term->term_id, 'languages' );
```

Comment:
@jtsternberg - purely an excuse, but during the time of writing this I was using sublime, so it didn't complain about much - if I were using PHPStorm this wouldn't have happened :smile: 

Don't judge me lol